django/db/backends/oracle/utils.py

```diff
@@ -32,7 +33,9 @@ def __init__(self, field):
         self.bound_param = None
 
     def bind_parameter(self, cursor):
-        self.bound_param = cursor.cursor.var(self.db_type)
+        self.bound_param = cursor.cursor.var(
+            self.db_type, arraysize=cursor.cursor.arraysize
```

Comment:
Do you think is worth to add some sort of comment here explaining why `arraysize` is defined? Following the docs

> The arraysize parameter specifies the number of elements the variable will have. If not specified the bind array size (usually 1) is used. When a variable is created in an output type handler this parameter should be set to the cursor’s array size.

perhaps we could add something like?

`arraysize should be set the cursor’s array size to allow for proper number of elements in the variable created by the output type handler`

Reply:
It's not created in an output type handler, so we can remove it.